Source/Fuse.Elements/Drawing/ImageFill.uno

```diff
@@ -110,7 +170,8 @@ namespace Fuse.Drawing
 			public float2 Origin, Size;
 			public float4 UVClip;
 			public Texture2D Texture;
-			public float2 TexCoordBias1, TexCoordBias2, TexCoordScale1, TexCoordScale2;
+			public float2 TexCoordBias1, TexCoordBias2, TexCoordScale1, TexCoordScale2, TexCoordOffset, ScaleOrigin;
+			public float ScaleFactor;
```

Comment:
Maybe, I didn't want to intrude on things I didn't entirely understand and keep the pr features as separate as I could